force-app/main/default/classes/Homework.cls

```diff
@@ -1,60 +1,79 @@
 public with sharing class Homework {
     // Declare a variable to hold the postcode of Ginetta House - as
     // the office cannot move, this should be immutable
+    static final String postCode = 'WS13 8XG';
 
 
     // Declare a string that is initialised from an Integer
-
+    
+    String stringValueOfnumberOfStudents = String.valueOf('20');
 
     public void newYearsEve()
     {
         // Create a variable representing the time 12:30:25 in the local 
         // timezone and use this time to create a DateTime for the date 
         // 12/31/2021
+
+        Time localTime = Time.newInstance(12, 30, 25, 0);
+        Datetime myNewYearEveDateTime = Datetime.NewInstance(2021, 12, 31, 12, 30, 25);
```

Comment:
This is not correct - the comment says to use the `localTime` variable when creating the new DateTime, whereas you have supplied the individual time components. You should be passing `localtime` as a parameter.